src/NuGet.Services.V3/Registration/Models/ICommitted.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace NuGet.Protocol.Registration
+{
+    public interface ICommitted
+    {
+        string CommitId { get; set; }
+        DateTimeOffset CommitTimestamp { get; set; }
```

Comment:
Why is the comittimestamp important? Is there an user scenario for it?

Reply:
There is no known user scenario around commit timestamp. That being said, it is currently written in today's blob's so having it in the model is required if we want to avoid any unintentional/unofficial protocol breaks.